install/kubernetes/istio-sidecar-initializer.yaml

```diff
@@ -0,0 +1,79 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-inject
+  namespace: istio-system
+data:
+  config: |-
+    policy: "enabled"
+    namespaces: [""] # everything, aka v1.NamepsaceAll, aka cluster-wide
+    initializerName: "sidecar.initializer.istio.io"
+    params:
+      initImage: gcr.io/istio-testing/proxy_init:5acaa9e26e7d9b239cbc72da63f5f9a3dac5a54e
+      proxyImage: gcr.io/istio-testing/proxy:5acaa9e26e7d9b239cbc72da63f5f9a3dac5a54e
+      verbosity: 2
+      version: 5acaa9e26e7d9b239cbc72da63f5f9a3dac5a54e
+      meshConfigMapName: istio
+      imagePullPolicy: IfNotPresent
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: istio-sidecar-initializer-service-account
+  namespace: istio-system
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: istio-sidecar-initializer
+  namespace: istio-system
+  annotations:
```

Comment:
Do we really need the annotation here? We already defined the `initializers` as `[]`, so seems here do not need the annotations.

```
initializers:
  pending: []
```